stripe/src/main/java/com/stripe/android/net/PollingParameters.java

```diff
@@ -0,0 +1,75 @@
+package com.stripe.android.net;
+
+/**
+ * Container class for polling parameters.
+ */
+class PollingParameters {
+
+    private static final long DEFAULT_TIMEOUT_MS = 10000L;
+    private static final long INITIAL_DELAY_MS = 1000;
+    private static final long MAX_DELAY_MS = 15000;
+    private static final int MAX_RETRY_COUNT = 5;
+    private static final long MAX_TIMEOUT_MS = 5L * 60L * 1000L; // five minutes
+    private static final int POLLING_MULTIPLIER = 2;
+
+    private final long mDefaultTimeoutMs;
+    private final long mInitialDelayMs;
+    private final long mMaxDelayMs;
+    private final int mMaxRetryCount;
+    private final long mMaxTimeoutMs;
+    private final int mPollingMultiplier;
+
+    PollingParameters(
```

Comment:
Not a public class, so no Javadoc (though I have added it for other package-private ones, but it's not a part of the public-facing api)